tests/system/providers/google/cloud/datafusion/example_datafusion_async.py

```diff
@@ -145,19 +146,19 @@
 # [END howto_data_fusion_env_variables]
 
 with models.DAG(
-    "example_data_fusion_async",
+    DAG_ID,
     start_date=datetime(2021, 1, 1),
     catchup=False,
```

Comment:
Could you please add tags into both dags?

ref: https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-47+New+design+of+Airflow+System+Tests
"Process of migration in details" -> section 5.b

Reply:
Sure, thank you!